extras-protobuf/src/main/scala/com/avast/clients/rabbitmq/extras/format/ProtobufConsumer.scala

```diff
@@ -0,0 +1,16 @@
+package com.avast.clients.rabbitmq.extras.format
+
+import com.avast.bytes.Bytes
+import com.avast.clients.rabbitmq.MultiFormatConsumer
+import com.avast.clients.rabbitmq.api.{Delivery, DeliveryResult}
+import com.google.protobuf.GeneratedMessageV3
+
+import scala.language.higherKinds
+import scala.reflect.ClassTag
+
+object ProtobufConsumer {
+  def create[F[_], A <: GeneratedMessageV3: ClassTag](action: Delivery[A] => F[DeliveryResult]): (Delivery[Bytes] => F[DeliveryResult]) =
```

Comment:
This is V3 only?

Reply:
Yes, because it is IMHO better to make the restriction to _ensure_ that the `static` `newBuilder` method exists.

We could add another method for older Protobuf messages if required but I would prefer not to add it as we don't need it right now.